server/portal/templates/includes/navigation.html

```diff
@@ -1,96 +1,55 @@
+        </a>
+
+
+
+
 </div>
 
+
+
 <!-- Navigation Bar -->
 <nav id="s-header" class="s-header  navbar navbar-expand-lg navbar-dark">
-  <!-- Branding Logos -->
-  <a class="navbar-brand" href="/">
-    <img class="nav-logo-double nav-logo-frontera-double" src="/static/site_cms/images/org_logos/frontera-white-solo.png" crossorigin="anonymous">
-  </a>
+  <!-- Portal Logo -->
+
+
+    <a class="navbar-brand" href="/" target="">
+    <img class="portal-logo " src="/static/frontera-cms/img/org_logos/frontera-white-solo.png" crossorigin="anonymous" alt="Frontera Logo">
+    </a>
+
+
+
   <!-- Navbar Accordian Toggle on Small Screens  -->
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExpandTarget" aria-controls="navbarsExpandTarget" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
+
   <!-- Navbar Links -->
   <div class="collapse navbar-collapse" id="navbarsExpandTarget">
-    <ul class="navbar-nav" id="cms-nav-pages" data-cms-nav-endpoint="/cms/nav/pages/markup/">
-      <!-- All top-level CMS controlled nav elements handled here. -->
-    </ul>
-
-    {# Duplicated at https://gitlab.tacc.utexas.edu/wma-cms/frontera-cms/blob/master/taccsite_cms/templates/nav_branding_partials/nav.branding.frontera.v2.html #}
-    {# Duplicated at https://bitbucket.org/taccaci/frontera-tech-docs/src/staging/frontera_theme/header.html #}
-    <ul class="s-portal-nav  navbar-nav navbar-right  ml-auto">
-      {% if user.is_authenticated %}
-      <div class="navbar-nav">
-        <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            <i class="fas fa-user-circle fa-lg nav-icon"></i> {{ user.get_username }}
-            <span class="sr-only">Toggle Dropdown</span>
-          </a>
-          <div class="dropdown-menu dropdown-menu-right">
-            <a class="dropdown-item" href="{% url 'workbench:dashboard' %}">
-              <i class="fas fa-desktop nav-icon"></i> My Dashboard
-            </a>
-            <a class="dropdown-item" href="{% url 'workbench:account' %}">
-              <i class="fas fa-user-circle nav-icon"></i> My Account
-            </a>
-            <a class="dropdown-item" href="{% url 'portal_accounts:logout' %}">
-              <i class="fas fa-sign-out-alt nav-icon"></i> Log Out
-            </a>
-          </div>
-        </li>
-      </div>
-      {% else %}
-      <li class="nav-item">
-        <a class="nav-link" role="button" href="{% url 'portal_auth:agave_oauth' %}">
-          <i class="fas fa-sign-in-alt nav-icon"></i> Log in
-        </a>
-      </li>
-      {% endif %}
-      <!-- end of user logged in/not logged in if -->
-    </ul>
+    {# FAQ: Only the version of this template _from the CMS repo_ can not use `only`; here can #}
```

Comment:
__Explanation__: This is a reminder to developers who reference [similar code in Core CMS](https://gitlab.tacc.utexas.edu/wma-cms/cms-site-template/blob/master/taccsite_cms/templates/branding.html#L51). A comment at the top of this file points to a document which illiustrates this connection.